components/PointerBox/PointerBox.tsx

```diff
@@ -2,17 +2,18 @@ import React from 'react'
 import Arrow from '../Arrow/Arrow';
 
 type Props = {
-    color?: number[];
-    pointed?: boolean;
-    arrow: boolean;
+    color?: number[]
+    pointed?: boolean
+    arrow: boolean
+    arrowOrientation: string
 }
 // TODO: Fix colors
-function PointerBox({ pointed, arrow=false, color = [122, 123, 124] }: Props) {
+function PointerBox({ color = [122, 123, 124], pointed=false, arrow=false, arrowOrientation='right'}: Props) {
```

Comment:
format documents before pushing
maybe set autoformating on save in VSC
otherwise look for a shortcut